code/game/machinery/constructable_frame.dm

```diff
@@ -754,6 +754,15 @@ obj/item/weapon/circuitboard/rdserver
 							"/obj/item/weapon/stock_parts/matter_bin" = 1,
 							"/obj/item/weapon/stock_parts/capacitor" = 2)
 
+/obj/item/weapon/circuitboard/box_cloner
+	name = "Circuit Board (Box Cloner)"
+	build_path = "/obj/machinery/egg_incubator/box_cloner"
+	origin_tech = "illegal=3"
```

Comment:
Weird, you're right. I'll change it to syndicate.

I'm not too concerned about a savvy mechanic getting some research by trading. That actually seems like a pretty good idea, trading for valuable research items.